src/stores/index.js

```diff
@@ -0,0 +1,16 @@
+const redux = require('redux');
+const reducers = require('../reducers');
+
+module.exports = function(initialState) {
+  const store = redux.createStore(reducers, initialState)
```

Comment:
Nah, we don't at this time; I will remove it. It can be added in a future story.